app/controllers/concerns/sign_in_audit_concerns.rb

```diff
@@ -12,20 +12,15 @@ def sign_in_event(status:)
     MessageEncryptor.new(data).encrypt
   end
 
-  def log_succesful_authentication
+  def audit_successful_authentication
     Auditor::Audit.new(nil, 'dfe-sign-in.authentication.success', current_session_id).log_without_association
   end
 
-  def log_succesful_authorisation
+  def audit_successful_authorisation
     Auditor::Audit.new(current_school, 'dfe-sign-in.authorisation.success', current_session_id).log
   end
 
-  def log_failed_authorisation
+  def audit_failed_authorisation
     Auditor::Audit.new(nil, 'dfe-sign-in.authorisation.failure', current_session_id).log_without_association
-    Rails.logger.warn("Hiring staff not authorised: #{oid} for school: #{selected_school_urn}")
-  end
-
-  def user_dsi_id
```

Comment:
Does this get used in the method above?  ` sign_in_event ` ?

Reply:
good catch, added it back in https://github.com/DFE-Digital/teacher-vacancy-service/pull/862/commits/aa8dd5dec00d3f88aee7bed423db8191d0ba5a1b